docs/reference/changelog.rst

```diff
@@ -70,6 +70,102 @@ Changelog
 
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
+8.5.0 (2024-Nov-19)
+-------------------
+
+Core
+====
+
+* Remove the Snapcraft ``Dockerfile`` in favour of the `snapcraft-rocks`_
+  registry. For more information, see the ``docker/README.md`` file.
+
+Bases
+#####
+
+core24
+""""""
+
+* Add a ``--platform <platform>`` argument to the ``clean`` to filter
+  which build environments to clean (`canonical/craft-application#425`_).
+
+Plugins
+#######
+
+Poetry
+""""""
+
+* Add a new plugin for projects using `Poetry <https://python-poetry.org/>`_.
+  For more information, see :doc:`Poetry plugin
+  reference</reference/plugins/poetry_plugin>`.
+
+Extensions
+##########
+
+Env-injector
+""""""""""""
+
+* Add a new experimental extension called
+  `env-injector <https://github.com/canonical/snappy-env>`_. It simplifies
+  passing environment variables to snaps at runtime using ``snap config`` or a
+  ``env`` file. For more information, see `the env-injector
+  extension <https://forum.snapcraft.io/t/the-env-injector-extension/41477>`_
+
+Command line
+============
+
+* Order lifecycle commands in the sequence they run rather than alphabetically
+  in help messages.
+
+* Show positional arguments in help messages.
+
+* Hide the terminal cursor when Snapcraft is running.
+
+* Fix a bug where the version and path to the log file were logged multiple
+  times.
+
+* Fix a bug where deprecated commands were suggested when an invalid command
+  was entered (`canonical/craft-cli#297`_).
+
+Init
+====
+
+* Add a ``<project-dir>`` argument to initialise the project in a particular
+  directory.
+
+* Add a ``--name <name>`` argument to set the ``name`` key in the initialised
+  ``snapcraft.yaml``. If ``--name`` is not provided, the name of the project
+  directory or current working directory is used.
+
+* Add a ``--profile <profile>`` argument to initialise a project for a specific
+  use-case. Currently, only the ``simple`` profile is supported.
```

Comment:
```suggestion
  purpose. Currently, only the ``simple`` profile is supported.
```

"Use case" is on our [list of deprecated phrases](https://docs.ubuntu.com/styleguide/en/#words-and-phrases-to-avoid).